scipy/special/tests/test_cdflib.py

```diff
@@ -548,8 +547,11 @@ def test_ncfdtr(dfn, dfd, nc, f, expected):
     # rng = np.random.default_rng(1234)
     # sample_idx = rng.choice(len(re), replace=False, size=12)
     # cases = np.array(cases)[sample_idx].tolist()
-    assert_allclose(sp.ncfdtr(dfn, dfd, nc, f), expected, rtol=1e-13, atol=0)
-
+    assert_allclose(sp.ncfdtr(dfn, dfd, nc, f), expected_cdf, rtol=1e-13, atol=0)
+    # testing tails where the CDF reaches 0 or 1 does not make sense for inverses
+    # of a CDF as they are not bijective in these regions
+    if 0 < expected_cdf < 1:
+        assert_allclose(sp.ncfdtri(dfn, dfd, nc, expected_cdf), f, rtol=5e-11)
```

Comment:
I suppose we could wait for Albert to comment, but given that no algorithmic code is changing and the numerical stringency of the original testing remains the same (just a new test case is added)--this is probably pretty safe in any case.

I looked at the few CI failures--they're all unrelated and were recently fixed in `main`.